tests/unit/utils/bind-data-test-attributes-test.js

```diff
@@ -1,24 +1,27 @@
 import { module, test } from 'qunit';
 import EmberObject, { computed } from '@ember/object';
+import config from 'dummy/config/environment';
 
 import bindDataTestAttributes from 'ember-test-selectors/utils/bind-data-test-attributes';
 
 module('Unit | Utility | bind data test attributes');
 
 test('it adds missing attributeBindings array', function(assert) {
   let Fixture = EmberObject.extend({
-    'data-test-from-factory': 'foo',
+    'data-test-from-factory': 'foo'
```

Comment:
can you remove these unrelated changes? the PR should only contain the changed functionality. Styling changes should be proposed in a seperate PR.